src/services/qrProcessor/qrProcessor.test.js

```diff
@@ -1,39 +1,64 @@
 import { decodeQrCode, processQrCode } from "./index";
 
 const encodeQrCode = (payload) => `https://action.openattestation.com?q=${encodeURIComponent(JSON.stringify(payload))}`;
+const encodeQrCodeWithAnchor = (payload, anchor) =>
+  `https://action.openattestation.com?q=${encodeURIComponent(JSON.stringify(payload))}#${encodeURIComponent(
+    JSON.stringify(anchor)
+  )}`;
 
 describe("decodeQrCode", () => {
   it("decodes an action correctly", () => {
-    const encodedQrCode =
-      "https://action.openattestation.com?q=%7B%22uri%22%3A%22https%3A%2F%2Fsample.domain%2Fdocument%2Fid%3Fq%3Dabc%23123%22%7D";
+    const sampleAction = {
+      uri: "https://sample.domain/document/id?q=abc#123",
+    };
+
+    const { action, anchor } = decodeQrCode(encodeQrCode(sampleAction));
+    expect(action).toStrictEqual(action);
```

Comment:
this will pass regardless? is it supposed to be `expect(action).toStrictEqual(sampleAction);`

Reply:
You're right, made a typo here. Will fix it.